tests/integration/targets/azure_rm_networkinterface/tasks/main.yml

```diff
@@ -319,6 +322,89 @@
       - output.state.ip_configurations[0].public_ip_address.name == "tn{{ rpfx }}"
       - output.state.enable_accelerated_networking
 
+- name: Create Application security group
+  azure_rm_applicationsecuritygroup:
```

Comment:
as discussed, maybe remove standalonge asg test and just have this test here.

Reply:
fixed